decoders/ps2/pd.py

```diff
@@ -42,18 +54,27 @@ class Decoder(srd.Decoder):
     annotations = (
         ('bit', 'Bit'),
         ('start-bit', 'Start bit'),
-        ('stop-bit', 'Stop bit'),
+        ('word', 'Word'),
         ('parity-ok', 'Parity OK bit'),
         ('parity-err', 'Parity error bit'),
-        ('data-bit', 'Data bit'),
+        ('stop-bit', 'Stop bit'),
+        ('ack', 'Acknowledge'),
+        ('nack', 'Not Acknowledge'),
+        ('start-bit', 'Start bit'),
         ('word', 'Word'),
+        ('parity-ok', 'Parity OK bit'),
+        ('parity-err', 'Parity error bit'),
     )
     annotation_rows = (
         ('bits', 'Bits', (0,)),
-        ('fields', 'Fields', (1, 2, 3, 4, 5, 6)),
+        ('fields', 'Device', (1,2,3,4,5,6,7,)),
+        ('host', 'Host', (8,9,10,11,)),
     )
 
     def __init__(self):
+        if 0:
```

Comment:
is this block left in on purpose ?

Reply:
Oops. I will remove the debugger initialization.